robinklaassen/aas2
# Da email integration

Update  email list based on active members

# How to test

- Ask vincent for `.env`  for the propper credentials
- Update or regisiter a `member`
- Check subscribers of target list in `DirectAdmin` 
- See it updated
